Configuration/Geometry/python/GeometryDDDSimDB_cff.py

```diff
@@ -8,8 +8,6 @@
 from Geometry.MuonNumbering.muonNumberingInitialization_cfi import *
 from Geometry.TrackerNumberingBuilder.trackerNumberingGeometryDB_cfi import *
 from Geometry.EcalCommonData.ecalSimulationParameters_cff   import *
-from Geometry.HcalCommonData.hcalSimulationParameters_cff   import *
-from Geometry.HcalCommonData.hcalDDDSimConstants_cfi        import *
-from Geometry.HcalCommonData.hcalSimulationConstants_cfi    import *
-from Geometry.HcalCommonData.caloSimulationParameters_cff   import *
+from Geometry.HcalCommonData.hcalSimDBConstants_cff         import *
 from Geometry.MuonNumbering.muonGeometryConstants_cff       import *
+from Geometry.MuonNumbering.muonOffsetESProducer_cff        import *
```

Comment:
Why is the `muonOffsetESProducer_cff` being added? The DDD workflows ran successfully without it, so why is it needed?

Reply:
It is strictly not needed - I just wanted to symmetrize between ddd and dd4hep parts. It also does not cause harm.

Reply:
Please add a comment to the file saying that `muonOffsetESProducer_cff` is not needed but is included for consistency with the DD4hep version.

Reply:
Added